backend-rust/src/connection.rs

```diff
@@ -72,26 +77,26 @@ impl ConnectionCursor for i64 {
 pub struct DescendingI64(i64);
 
 impl ConnectionCursor for DescendingI64 {
-    const MAX: Self = Self(i64::MIN);
-    const MIN: Self = Self(i64::MAX);
+    const END_BOUND: Self = Self(i64::MIN);
+    const START_BOUND: Self = Self(i64::MAX);
 }
 
 /// Prepared query arguments for SQL query, based on arguments from a GraphQL
 /// Cursor Connection resolver.
 #[derive(Debug)]
 pub struct ConnectionQuery<A> {
     /// The lower to use for the SQL query.
-    pub from:  A,
+    pub from:    A,
     /// The upper to use for the SQL query.
-    pub to:    A,
+    pub to:      A,
     /// The limit to use for the SQL query.
-    pub limit: i64,
+    pub limit:   i64,
     /// If the `last` elements are requested instead of the `first` elements
     /// (indicated by the `last` key being set when creating a new
     /// `ConnectionQuery`), the edges/nodes should be ordered in reverse
     /// (DESC) order before applying the range. This allows the range from
     /// `from` to `to` to be applied starting from the last element.
```

Comment:
```suggestion
    /// `ConnectionQuery`), the edges/nodes should be ordered in reverse
    /// order to the returned order of the connection before applying the range. This allows the range from
    /// `from` to `to` to be applied starting from the last element.
```

Similar comments exist on some of the queries using this boolean.

Reply:
We can fix them as we find them :smile: